content/contribute/add_phone_to_plugin.md

```diff
@@ -0,0 +1,234 @@
+# Add a phone model to an existing plugin
```

Comment:
That's a brutal start. How about an introduction.

In this section we are going to see how to blah blah blah...